modules/cco-ccoctl-upgrading.adoc

```diff
@@ -0,0 +1,89 @@
+//Test plan has a link to several health checks, but what specifically should the user run and what are they watching out for?
+
+. Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory:
++
+[source,terminal,subs="+quotes"]
+----
+$ ccoctl aws create-iam-roles --name __<name>__ --region=__<aws_region>__ --credentials-requests-dir=__<path_to_directory_with_list_of_credentials_requests>__/credrequests --identity-provider-arn arn:aws:iam::__<aws_account_id>__:oidc-provider/__<name>__-oidc.s3.__<aws_region>__.amazonaws.com
+----
++
+where:
++
+** _<name>_ is the name used to tag any cloud resources that are created for tracking.
+** _<aws_account_id>_ is the AWS account ID.
+** _<aws_region>_ is the AWS region in which cloud resources will be created.
++
+[NOTE]
+====
+For AWS environments that use alternative IAM API endpoints, such as GovCloud, you must also specify your region with the `--region` parameter.
+====
++
+For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a trust policy that is tied to the specified OIDC identity provider, and a permissions policy as defined in each `CredentialsRequest` object from the {product-title} release image.
+
+. Add any new permissions in the appropriate role and policy on AWS.
+//At this point in the test plan, it is stated that "there are 2 new permissions added for machine-api." How does the user figure this out? Would output from the previous step help?
+//Not really clear on the correct thing to tell the user to do on the AWS side, what is this process called? Any AWS docs we should refer them to?
++
+It may take up to several minutes for the updated permissions to take effect.
+
+. Complete appropriate health checks. #elaborate#
+//Again, test plan has a link to several health checks, but what specifically should the user run and what are they watching out for?
+
+.Verification
+
+//Does this make sense for the upgrade as well?
+* To verify that the {product-title} secrets are created, list the files in the `_<path_to_ccoctl_output_dir>_/manifests` directory:
++
+[source,terminal,subs="+quotes"]
+----
+$ ll __<path_to_ccoctl_output_dir>__/manifests
+----
++
+.Example output:
++
+[source,terminal,subs="+quotes"]
+----
+total 24
+-rw-------. 1 __<user>__ __<user>__ 161 Apr 13 11:42 cluster-authentication-02-config.yaml
+-rw-------. 1 __<user>__ __<user>__ 379 Apr 13 11:59 openshift-cloud-credential-operator-cloud-credential-operator-iam-ro-creds-credentials.yaml
+-rw-------. 1 __<user>__ __<user>__ 353 Apr 13 11:59 openshift-cluster-csi-drivers-ebs-cloud-credentials-credentials.yaml
+-rw-------. 1 __<user>__ __<user>__ 355 Apr 13 11:59 openshift-image-registry-installer-cloud-credentials-credentials.yaml
+-rw-------. 1 __<user>__ __<user>__ 339 Apr 13 11:59 openshift-ingress-operator-cloud-credentials-credentials.yaml
+-rw-------. 1 __<user>__ __<user>__ 337 Apr 13 11:59 openshift-machine-api-aws-cloud-credentials-credentials.yaml
+----
```

Comment:
This section doesn't really make sense or add much value for the upgrade usecase.